__tests__/commands/authError.test.ts

```diff
@@ -0,0 +1,77 @@
+import { describe, expect, test } from '@jest/globals';
+import { ERRORS } from '../../src/constant';
+import {
+  appIDValidator,
+  authValidator,
+  domainValidator,
+  passwordValidator,
+  proxyValidator,
+  usernameValidator
+} from '../../src/commands/Auth/validator';
+
+describe('auth command: validator', () => {
+  test(`The name app missing -> ${ERRORS.APP_NAME_MISSING}`, () => {
+    const input = '';
+    expect(authValidator(input)).toBe(ERRORS.APP_NAME_MISSING);
+  });
+
+  test(`Domain validator: missing https -> ${ERRORS.DOMAIN_STARTS_WITH_HTTPS}`, () => {
+    const input = 'domain.kintone.com';
+    expect(domainValidator(input)).toBe(ERRORS.DOMAIN_STARTS_WITH_HTTPS);
+  });
+
+  test(`Domain validator: valid domain -> ${ERRORS.VALID_DOMAIN}`, () => {
```

Comment:
Please improve the description to be able to know what is the case?
With the current description, I can know ERRORS.VALID_DOMAIN will be outputted when setting valid domain.
I think it's wrong.